### ISSUE TITLE
Mangagamer copy not working?

### ISSUE THREAD

**Issue by shiropan** [higurashi-mod/onikakushi#36]
Hi. I have the Mangagamer copy of the game. Based upon the file structure it looks like everything is the same, so hopefully you can help me out. I followed the full PS3 voices and sprites patch. However I still am not getting any voices when I play the game. Here is how I installed it:
>Installed HigurashiPS3-Voices01.zip, moved all files to SE folder
>Downloaded PS3 sprites 1.5 and moved CGAlt into CGAlt folder
>Downloaded PS3 Voice and Graphics patch and moved SE and Update folder into StreamingAssets folder
>Started the game
Just not getting any voices... PS3 sprites appear, etc. Any help?

**Comment by ItaloKnox** [comment]
The patch is confirmed to work with the MangaGamer version, so I guess you have a problem with the CompiledScripts folder. Take a look at [this](https://github.com/higurashi-mod/onikakushi#troubleshooting) and report back.

Just one more thing: did you actually download the patch? [This patch](https://github.com/higurashi-mod/onikakushi/releases/tag/v.1.3), I mean.

**Comment by shiropan** [comment]
Reinstalled, tried the patch you linked, tried the CompiledScripts fix, still no voices. Crap.

**Comment by ItaloKnox** [comment]
That's bad, there must be something missing here. Can you install everything again? Starting from the game, just install everything before opening the game (to avoid caching the scripts). You can open it after installing the voices, graphics and the patch (in this order).

**Comment by shiropan** [comment]
Okay. I re-downloaded it again, but it looks like my old save that I made about 5 minutes in is still there, even though I deleted the entire old game folder. Is there AppData for the game or something?

**Comment by ItaloKnox** [comment]
Yes, you should have the Update folder in there, and then overwrite the files with the patch.

The save file is located at %appdata%\Mangagamer. Delete this folder (backup it first), then delete the contents of the CompiledScripts folder. Run the game. If it's still not working, redo the previous steps and look for a folder called CompiledUpdateScripts. Backup it first, then also delete its contents. Now run the game.

If your game stops working after deleting the stuff inside CompiledUpdateScripts, just restore your backup, delete what was created inside the CompiledScripts folder and then run the game again. Even after all that stuff your still don't get voices working, then delete only the content of the CompiledUpdateScripts folder and then run the game. Leave the CompiledScripts alone this time.

I'm not giving up on this but I don't have many options left. Something's really missing here.

**Comment by shiropan** [comment]
Okay, reinstalled, deleted AppData and CompiledScripts. Didn't work. 
Now where is CompiledUpdateScripts? 
![ss 2015-11-14 at 11 12 21](https://cloud.githubusercontent.com/assets/15841776/11167767/58afe2dc-8b25-11e5-8e5e-a6cdf26f59c8.png)

**Comment by Cnnctr** [comment]
So, I downloaded the game from MangaGamer, in case they changed anything. And it still works.

1) Have you really downloaded it from MG? If it's pirated, it might still be on version 1.00, which didn't support Update folder yet.
2) Check log for errors, it should be in HigurashiEp01_Data. Try to look for reason why it refuses to compile. Maybe try to run it as administrator.

**Comment by ItaloKnox** [comment]
The CompiledUpdateScripts feature was added in later patches. I guess you are running an early version. It should work on both versions, so no problem here.

I had this same problem today when I was working on EP2's patch. Here's how I solved it, might work for you too:

1- delete everything inside CompiledScripts;
2- delete your Update folder. Duplicate the Scripts folder and rename it to Update;
3- run the game;
4- if you got to the main menu without a problem, close the game, repeat step 1;
5- [now install the patch](https://github.com/higurashi-mod/onikakushi/archive/master.zip).

Sometimes the engine just refuses to recompile the scripts. No matter how many times you move things around, just experienced this.
